content/02.introduction.md

```diff
@@ -18,51 +18,51 @@ humans [@doi:10.1038/s41586-018-0094-2; @doi:10.1126/science.1224344;
+have led to a heterogeneous landscape in terms of the types of analyses undertaken, 
+and their computational resource requirements [@doi:10.1093/bib/bbu022;
 @doi:10.3389/fgene.2018.00575]. Taking into consideration the unequal
 distribution of resources (and infrastructure such as internet connection),
 streamlined and efficient pipelines can help increase accessibility to
-high-quality analyses
+high-quality analyses.
 
 The degraded nature of aDNA poses an extra layer of complexity to standard
 modern genomic analysis. Through a variety of processes [@doi:10.1038/362709a0],
-DNA molecules will fragment overtime resulting in ultra-short molecules
-[@doi:10.1038/nature17405]). These sequences have low nucleotide complexity
+DNA molecules fragment over time, resulting in ultra-short molecules
+[@doi:10.1038/nature17405]. These sequences have low nucleotide complexity
 making it difficult to identify with precision which part of the genome a read
-is derived from. Furthermore, when fragmentation is not a 'clean break' this can
-lead to uneven ends with single-stranded 'overhangs' at end of molecules, which
-are susceptible to chemical processes such as deamination that lead to
-mis-incorporation of bases during library construction
+is derived from. Furthermore, when this fragmentation is not a 'clean break' it can
+lead to uneven ends with single-stranded 'overhangs' at end of DNA molecules. These 
+overhangs are susceptible to chemical processes, such as deamination, that lead 
+to mis-incorporation of bases during genomic library construction
 [@doi:10.1073/pnas.0704665104]. On top of this, taphonomic processes such as
-heat, moisture, and microbial and burial environment processes lead to varying
+heat, moisture, and microbial- and burial-environment processes lead to varying
 rates of degradation [@doi:10.1093/nar/gkx361;
 @doi:10.1146/annurev-genom-091416-035526], where the original DNA content of a
-sample is lost and supplanted by modern environmental DNA. Later handling by
-archaeologists, museum curators, and scientists can also contribute 'modern'
-contamination. While these characteristics can help provide evidence towards the
-'authenticity' of true aDNA sequences (e.g. aDNA C>T 'damage' profiles
-[@doi:10.1093/bioinformatics/btt193]), they also pose specific challenges such
-as unspecific DNA alignment and/or low coverage and miscoding lesions resulting
-in low-confidence genotyping. These factors often lead to prohibitive sequencing
-costs to retrieve enough data for modern NGS data pipelines (e.g. > 1 billion
-reads for a 1X depth coverage _Yersinia pestis_ genome
-[@doi:10.1016/j.cell.2015.10.009]), and thus require aDNA tailored methods and
-techniques to overcome these challenges.
+sample is lost and supplanted by present-day environmental DNA. Later handling by
+archaeologists, museum curators, and specialists responsible for the laboratory work can also contribute traces of 
```

Comment:
```suggestion
archaeologists, museum curators, and scientists can also contribute traces of 
```
Bioinformatician bias there ;). I still collected samples and handled them during sampling before sent to the lab.

Reply:
If you don't like differentitation of archaeologists vs specialists, could say archaeogeneticists instead.

Reply:
I don't like the differentiation between archaeologists and scientists.
Archaeogeneticists would work